.github/workflows/release.yml

```diff
@@ -16,21 +16,17 @@ jobs:
     permissions:
       id-token: write
     steps:
-      - name: Setup python to build package
-        uses: actions/setup-python@v6
+      - name: Install uv & Python
+        uses: astral-sh/setup-uv@v6
         with:
-          python-version: '3.12'
-      - name: Install build
-        run: python -m pip install build
+          python-version: "3.13"
       - uses: actions/checkout@v5
         with:
           fetch-depth: 0
       - name: Build ext package
-        run: python -m build ext/
+        run: uv build --package=django-stubs-ext
```

Comment:
* Oops -- this was omitted in the previous PR https://github.com/typeddjango/django-stubs/pull/2848